src/main/java/com/dev/bankingservice/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.dev.bankingservice.service.impl;
+
+import com.dev.bankingservice.entity.User;
+import com.dev.bankingservice.repository.UserRepository;
+import com.dev.bankingservice.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class UserServiceImpl implements UserService {
+    private final UserRepository userRepository;
+
+    @Override
+    public User create(User user) {
+        return userRepository.save(user);
+    }
+
+    @Override
+    public User getById(Long id) {
+        return userRepository.getById(id).orElseThrow(()
+                -> new RuntimeException("Can not get user by id: " + id));
+    }
+
+    @Override
+    public User getByPhoneNumber(String phoneNumber) {
+        return userRepository.getByPhoneNumber(phoneNumber).orElseThrow(()
+                -> new RuntimeException("Can not get user by phone number: " + phoneNumber));
+    }
+
+    @Override
+    public void update(User user) {
+        userRepository.save(user);
+    }
+
+    @Override
+    public void delete(Long id) {
+        userRepository.delete(userRepository.getOne(id));
```

Comment:
```suggestion
        userRepository.deleteById(id);
```

Reply:
Done

Reply:
probably you forgot to push the change here?

Reply:
Certainly, fixed it now